src/cc/BPF.h

```diff
@@ -90,6 +94,11 @@ class BPF {
 
   template <class KeyType, class ValueType>
   BPFHashTable<KeyType, ValueType> get_hash_table(const std::string& name) {
+    if (ts_) {
+      TableStorage::iterator it;
+      if (ts_->Find(Path({name}), it))
+        return BPFHashTable<KeyType, ValueType>(it->second);
+    }
     return BPFHashTable<KeyType, ValueType>(bpf_module_.get(), name);
```

Comment:
Well, neither the new or existing logics handle cases where `name` is an invalid key, so this function needs some further changes.